packages/definitions-parser/test/git.test.ts

```diff
@@ -35,12 +35,7 @@ testo({
     ).toEqual({ errors: ["Please delete all files in jest when adding it to notNeededPackages.json."] });
   },
   async tooManyDeletes() {
-    expect(await getNotNeededPackages(allPackages, [{ status: "D", file: "oops.txt" }])).toEqual({
```

Comment:
we want to prevent people from deleting, say, DT's own tslint.json or package.json. This makes it look like it's now legal.

Reply:
Sure, but how do I distinguish that change from removing a non-critical file, like a script or a dead readme?

Reply:
It's probably OK to allow for now; human reviewers aren't likely to go for deletions like that (although the officejs reviewers have approved *modifications* to package.json it sounds like).